app/modules/site_settings/__init__.py

```diff
@@ -27,3 +28,4 @@ def init_app(app, **kwargs):
     from . import models, resources  # NOQA
 
     api_v1.add_namespace(resources.api)
+    register_prometheus_model(models.Taxonomy)
```

Comment:
This may be thw wrong change as I think the taxonomy functionality in the prometheus code should generate this. 
Happy to take feedback either way of which should be used.